packages/ponybench/_runner.pony

```diff
@@ -127,7 +133,9 @@ class val AsyncBenchContinue
 
   fun complete() =>
     let e = Time.nanos()
-    Time.perf_end()
+    ifdef x86 then
+      Time.perf_end()
+    end
```

Comment:
Can you explain these new guards? Does ponybench not work on other platforms?

Reply:
Ah... I see: https://github.com/ponylang/ponyc/blob/e69cb05147684578405c296909fdf0867e583acc/packages/time/time.pony#L136